src/components/AssetsFilters/AssetsFilters.test.jsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import Enzyme from 'enzyme';
 import { AssetsFilters, mapDispatchToProps } from './index';
 import { assetActions } from '../../data/constants/actionTypes';
+import { filtersInitial } from './../../data/reducers/assets';
 
 const mount = Enzyme.mount;
 
 const defaultProps = {
-  assetsFilters: {},
+  assetsFilters: { ...filtersInitial },
```

Comment:
I have changed the initial defaultProps for the AssetsFilters component to be filtersInitial, which is

{
   Audio: false,
   Code: false,
   Documents: false,
   Images: false,
   Other: false,
}